blazor/spreadsheet/getting-started-webapp.md

```diff
@@ -72,13 +72,13 @@ cd BlazorWebApp.Client
 
 N> For more information on creating a **Blazor Web App** with various interactive modes and locations, refer to this [link](https://blazor.syncfusion.com/documentation/getting-started/blazor-web-app?tabcontent=visual-studio-code#render-interactive-modes).
 
-## Install Syncfusion<sup style="font-size:70%">&reg;</sup> Blazor Spreadsheet and Themes NuGet in the App
+## Install Blazor Syncfusion<sup style="font-size:70%">&reg;</sup> Blazor Spreadsheet and Themes NuGet in the App
```

Comment:
Need to remove Blazor, it seems multiple places we used Blazor.

Install Syncfusion Blazor Spreadsheet and Themes NuGet in the App is enough